app/views/application_drafts/index.html.slim

```diff
@@ -18,6 +19,12 @@ table.table.table-striped.table-bordered.table-condensed
   tbody
     - @application_drafts.each do |draft|
       tr
+        td
+          - if draft.position == 1
+            i.icon-star title='Preferred application'
+          - else
+            a href=prioritize_application_draft_path(draft) title='Make preferred' data-method='put'
```

Comment:
Will the `put` not be caught by the Rails UJS driver, firing an XHR? That way the user won't see the status change (the star icon). But I may be completely wrong on this :smiley_cat: